addons/mail/tests/common.py

```diff
@@ -1547,8 +1547,9 @@ def _filter_threads_fields(self, /, *threads_data):
         """ Remove store thread data dependant on other modules if they are not not installed.
         Not written in a modular way to avoid complex override for a simple test tool.
         """
-        if "rating.rating" not in self.env:
-            for data in threads_data:
+        pool = self.env["res.users"].pool  # any model works
```

Comment:
`pool` is actually the registry object so it should be equivalent to `self.env.registry`.

Reply:
Good to know, I was wondering if there was a more direct way to access it.